### PR TITLE
webpack2 docs and other updates

### PR DESCRIPTION
**Pull request by mparke**

### CODE REVIEW

File: webpack.config.prod.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ var _package = require('./package.json');
 _.devtool = 'cheap-module-source-map';
 _.entry = './src/index';
 _.plugins = [
-  new webpack.optimize.CommonsChunkPlugin('common.js'),
+  new webpack.optimize.CommonsChunkPlugin({
```
Comment (stvnjacobs) [inline webpack.config.prod.js:10]: @mparke is it ok to go from `common.js` to `commons.js` with no change elsewhere?

File: docs/webpack.config.dev.js [hunk 2]
```diff
@@ -36,33 +35,39 @@ module.exports = {
...
+              ]
+            }
+          }
+        ],
       },
       {
         test: /\.svg$/,
-        loaders: ['file'],
+        loader: ['file-loader'],
         include: path.join(__dirname, 'node_modules')
```
Comment (stvnjacobs) [inline docs/webpack.config.dev.js:69]: are we are keeping this here and yanking it in the root `webpack.config.dev.js`?

File: webpack.config.prod.js (hunk 1 above)
Comment (mparke) [reply webpack.config.prod.js:10]: good catch, will double check this

File: docs/webpack.config.dev.js (hunk 2 above)
Comment (mparke) [reply docs/webpack.config.dev.js:69]: is not currently needed in the root one I believe, but will double check

**Pull request merged**